src/utils.js

```diff
@@ -0,0 +1,144 @@
+const fs = require("fs")
+require("dotenv").config()
+const lockfile = require("proper-lockfile")
+const readline = require("readline")
+
+module.exports = {
+  fromAscii,
+  getRealmNetworkFromArgs,
+  getRealmNetworkFromString,
+  getWitnetArtifactsFromArgs,
+  getWitnetRequestMethodString,
+  isDryRun,
+  isNullAddress,
+  padLeft,
+  prompt,
+  readJsonFromFile,
+  overwriteJsonFile,
+  traceHeader,
+  traceTx,
+}
+
+function fromAscii (str) {
+  const arr1 = []
+  for (let n = 0, l = str.length; n < l; n++) {
+    const hex = Number(str.charCodeAt(n)).toString(16)
+    arr1.push(hex)
+  }
+  return "0x" + arr1.join("")
```

Comment:
We can achieve the same behaviour with `Array.from`
```suggestion
  return "0x" + Array.from(str, char => char.charCodeAt(0).toString(16)).join("");
```